CHANGELOG.md

```diff
@@ -2,7 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## TS CLI&SDK [2.1.0](https://github.com/marinade-finance/validator-bonds/compare/v2.0.6...v2.1.0) (2025-01-31)
+## TS CLI&SDK [2.1.2](https://github.com/marinade-finance/validator-bonds/compare/v2.1.0...v2.1.2) (2025-02-04)
```

Comment:
Flagging that neither of the tags `v2.1.0` and `v2.1.2` exists, and adding a note that I'm not sure if I should be checking this in future PRs, because afaik it's usually tagged only after merging/publishing.

Reply:
I updated the changelog a bit. Let's see if it is better now.